new-log-viewer/src/utils/data.ts

```diff
@@ -1,6 +1,87 @@
 import {Nullable} from "../typings/common";
 
 
+/**
+ * Checks if 'target' is bounded by the first and last value in a sorted array of numbers.
+ *
+ * @param data A number array sorted in ascending order.
+ * @param target
+ * @return `true` if is `target` is within bounds; `false` otherwise or when the array is empty.
```

Comment:
```suggestion
 * Checks if `target` is bounded by the first and last value in a sorted array of numbers.
 *
 * @param data An array sorted in ascending order.
 * @param target
 * @return Whether `target` is within the bounds of the array's values.
```

I think it's intuitive that a value can't exist within the bounds of an array if the array is empty, so I don't think we need to be explicit for that case.